packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -544,22 +612,72 @@ class SliverChildListDelegate extends SliverChildDelegate {
   /// The widgets to display.
   final List<Widget> children;
 
+  /// A map to cache key to index lookup for children
+  ///
+  /// _keyToIndex[null] is used as current index during the lazy loading process
+  /// in [_findChildIndex]. _keyToIndex should never be used for looking up null key.
+  final Map<Key, int> _keyToIndex;
+
+  bool get _isConstantInstance => _keyToIndex == null;
+
+  int _findChildIndex(Key key){
+    if(_isConstantInstance) {
```

Comment:
nit: space after if.

the rule is that you put no space between an identifier and an `(` if you are using the identifier to do a method call (e.g. `foo()`), and you _do_ put a space after a keyword if it is a keyword (like `for`, or `if` or whatever), because in that case you're not calling a method.

Reply:
fixed